man/coap_attribute.txt.in

```diff
@@ -0,0 +1,140 @@
+// -*- mode:doc; -*-
+// vim: set syntax=asciidoc,tw=0:
+
+coap_attribute(3)
+=================
+:doctype: manpage
+:man source:   coap_attribute
+:man version:  @PACKAGE_VERSION@
+:man manual:   libcoap Manual
+
+NAME
+----
+coap_attribute, coap_add_attr, coap_find_attr
+- work with CoAP attributes
+
+SYNOPSIS
+--------
+*#include <coap/coap.h>*
+
+*coap_attr_t *coap_add_attr(coap_resource_t *_resource_, const unsigned char 
+*_name_, size_t _nlen_, const unsigned char *_value_, size_t _vlen_, 
+int _flags_);*
+
+*coap_attr_t *coap_find_attr(coap_resource_t *_resource_, const unsigned 
+char *_name_, size_t _nlen_);*
+
+Link with -lcoap-1.
```

Comment:
We will run into trouble with burying the API number deep in the manual pages. How about something like

```
Link with  -lcoap-@LIBCOAP_API_VERSION@.
```
in the respective .in files?

Reply:
I had held off changing the -l entry as there is the potential that it could be -llibcoap-2-openssl, etc. as a set of alternatives if the library renaming patch goes ahead.

Easy enough to change it to -lcoap-@LIBCOAP_API_VERSION@ for now.

Reply:
Oh, you are right. Usually pkg-config solves this but, of course, there are many platforms that do not have pkg-config.

Reply:
Now -lcoap-@LIBCOAP_API_VERSION@in latest push